stdlib/public/core/CollectionOfOne.swift

```diff
@@ -68,38 +82,45 @@ extension CollectionOfOne: RandomAccessCollection, MutableCollection {
   /// The "past the end" position---that is, the position one greater than the
   /// last valid subscript argument.
   ///
-  /// In a `CollectionOfOne` instance, `endIndex` is always identical to
-  /// `index(after: startIndex)`.
+  /// In a `CollectionOfOne` instance, `endIndex` is always `1`.
   @inlinable // FIXME(sil-serialize-all)
   public var endIndex: Int {
     return 1
   }
   
-  /// Always returns `endIndex`.
+  /// Returns the position immediately after the given index.
```

Comment:
Any reason these get the generic definitions rather than the "gives a trivial answer" descriptions like the others?

Reply:
There's not really any other behavior that `startIndex` and `endIndex` can have, but here the "Always returns..." bit seemed too much like exposing implementation details.